packages/composer/test/helper.js

```diff
@@ -325,6 +330,47 @@ async function createComposer (t, composerConfig, logger = false) {
   return app
 }
 
+async function createComposerInRuntime (t, prefix, composerConfig) {
+  await mkdir(tmpBaseDir, { recursive: true })
+  const tmpDir = await mkdtemp(resolve(tmpBaseDir, prefix))
+  await mkdir(resolve(tmpDir, 'composer'), { recursive: true })
+  t.after(() => rm(tmpDir, { recursive: true, force: true }))
+
+  const composerConfigPath = resolve(tmpDir, 'composer/platformatic.composer.json')
+  const runtimeConfigPath = resolve(tmpDir, 'platformatic.runtime.json')
+
+  await writeFile(
+    runtimeConfigPath,
+    JSON.stringify({
+      $schema: 'https://platformatic.dev/schemas/v1.5.1/runtime',
```

Comment:
we have changed this pattern to use the schema store.

Reply:
Done